spago.dhall

```diff
@@ -0,0 +1,15 @@
+{ name = "psc-ide"
+, dependencies =
+  [ "aff"
+  , "arrays"
+  , "console"
+  , "effect"
```

Comment:
All the other dependencies were already specified in the Bower file, but this one is new. Spago added it when generating the config as it was included transitively, so I left it in place.